app/models/fund.py

```diff
@@ -1,11 +1,5 @@
 import inspect
 from dataclasses import dataclass
-from enum import Enum
```

Comment:
Are these not needed anymore?

Reply:
These are removed by Sarah. Basically hardcoded fund-short-names are removed here